### PR TITLE
Fix/profile avatar nullable

### PR DESCRIPTION
**Pull request by Jonathansoufer**

### CODE REVIEW

**Review by obernardovieira — COMMENTED** [review]
I guess I missed this before,
our design has icons. Use them (as svgs), avoid using 3rd party icons.

File: src/views/profile/auth/index.tsx [hunk 1]
```diff
@@ -283,7 +284,10 @@ function Auth() {
             >
                 <View
                     style={{
-                        height: Dimensions.get('screen').height * 0.45,
+                        height:
```
Comment (obernardovieira) [inline src/views/profile/auth/index.tsx:286]: I'm quite afraid it wont work in small screens :thinking:

Comment (obernardovieira) [reply src/views/profile/auth/index.tsx:286]: would scrollview be an option? We used it on the old login view

File: src/views/profile/index.tsx [hunk 2]
```diff
@@ -439,32 +443,31 @@ function ProfileScreen() {
                                 color={ipctColors.borderGray}
                                 style={{
                                     right:
-                                        -Dimensions.get('screen').height * 0.25,
+                                        -Dimensions.get('screen').height * 0.33,
                                     alignSelf: 'center',
                                 }}
                             />
```
Comment (obernardovieira) [inline src/views/profile/index.tsx:449]: I guess I missed this before,
our design has icons. Use them (as svgs), avoid using 3rd party icons.

**Review by obernardovieira — CHANGES_REQUESTED** [review]

File: src/views/profile/index.tsx (hunk 2 above)
Comment (Jonathansoufer) [reply src/views/profile/index.tsx]: Fixed both the icons and the dynamic spacing.

**Pull request merged**